libraries/listen.rb

```diff
@@ -34,8 +34,8 @@ def self.converted_listen_ports_and_addresses(node)
       return [] unless node['apache']['listen_ports'] || node['apache']['listen_addresses']
```

Comment:
I would expect it to return `[]` here. Unless only one of those listen_ports and listen_addresses are set, which seems weird to me. Since the apache2 cookbook `< 3.2` defaulted both of those arrays. https://github.com/svanzoest-cookbooks/apache2/blob/v3.1.0/attributes/default.rb#L269-L270